### DISCUSSION TITLE
公网服务器配置反向代理后https访问web页面正常但是无法挂载webdav

### DISCUSSION THREAD

**Discussion by chenyang137** [AlistGo/alist#6466] [Q&A]
公网服务器按照教程配置反向代理后https访问web页面正常访问，但是无法通过反向代理的https+域名挂载webdav

**Comment by hahach** [comment]
你不贴nginx之类的反代的配置，别人怎么帮你
另外，webdav的挂载 是 https://域名/dav  你看看有没有搞错

**Reply by hxp0618** [reply]
我也有这样的问题，我的反代配置如下，web页面可以通过https访问，但是webdav就不行，只能通过http://ip:port/dav来挂载，下面是我的图片
![image](https://github.com/alist-org/alist/assets/30236140/078c3a35-4dcf-4808-9e46-25acc0ff5d70)
![image](https://github.com/alist-org/alist/assets/30236140/e53fdc28-224d-4e15-a437-73d364f454dc)
![image](https://github.com/alist-org/alist/assets/30236140/732ccfef-44c1-4b3b-b5ba-d9123c27e02b)
![image](https://github.com/alist-org/alist/assets/30236140/78b54f1a-5011-44f4-bccd-9d865720546a)
<img width="404" alt="image" src="https://github.com/alist-org/alist/assets/30236140/86e1310d-7fdb-43d7-b734-3f099c1a8f4e">

![image](https://github.com/alist-org/alist/assets/30236140/cc8b0a6f-fbee-4c2d-85f1-f6494e95f3fa)



> 你不贴nginx之类的反代的配置，别人怎么帮你 另外，webdav的挂载 是 [https://域名/dav](https://%E5%9F%9F%E5%90%8D/dav) 你看看有没有搞错

**Reply by shalimujiang** [reply]
请问您弄好1panel的反向代理配置吗？我想看一看你正确弄完的配置可以吗？我也遇到了同样的问题。感谢

**Comment by Simonzhang8** [comment]
你怎么写了两个反向代理呀？（两个conf里各写了一个）
虽然不知道和这个原因有没有关系
